mscr-ui/src/common/components/side-navigation/index.tsx

```diff
@@ -169,7 +186,27 @@ export default function SideNavigationPanel({ user }: { user?: MscrUser }) {
             </MscrSideNavigationLevel2>
           ))}
         </MscrSideNavigationLevel1>
-      </MscrSideNavigation>
+            </MscrSideNavigation>
+          </div>
+        </div>
+        <FoldButtonWrapper className={'col-1 d-flex justify-content-center flex-column'} onClick={() => {
+        }}>
+          <Tooltip
+            title={isSidebarMinimized ? t('click-to-expand-sidebar') : t('click-to-minimize-sidebar')}
+            placement="top-end"
+            className={undefined}>
+            <FoldButton
+              aria-label="fold"
```

Comment:
Could be more descriptive, or you can leave it for when accessibility will be checked and improved later.